hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,24 +1,50 @@
 
 <a name="pi4"></a>
-## Raspberry Pi 4
-To enable USB mass storage boot on a Raspberry Pi 4:
+## Raspberry Pi 4B
 
-* Use the "Misc Utility Images" option in the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to create a SD card with the latest "Raspberry Pi 4 EEPROM boot recovery" image.
-* Update to Raspberry Pi OS 2020-08-20 or newer.
-* Use [raspi-config](../../../configuration/raspi-config.md) to choose between SD/USB (default) or SD/Network boot modes.
+To boot from a USB mass storage device, the Raspberry Pi 4B requires the bootloader EEPROM release from 3 September 2020 or later. Check which bootloader EEPROM version your Pi has using the `rpi-eeprom-update` tool:
 
-The full set of boot mode options is documented on the [bootloader configuration](../bcm2711_bootloader_config.md) page.
+```bash
+pi@raspi4b:~ $ sudo rpi-eeprom-update
+BCM2711 detected
+Dedicated VL805 EEPROM detected
+BOOTLOADER: up-to-date
+CURRENT: Thu  3 Sep 12:11:43 UTC 2020 (1599135103)
+ LATEST: Thu  3 Sep 12:11:43 UTC 2020 (1599135103)
+ FW DIR: /lib/firmware/raspberrypi/bootloader/critical
+VL805: up-to-date
+CURRENT: 000138a1
+ LATEST: 000138a1
+ ```
+ 
+The `rpi-eeprom-update` tool reports information about two different EEPROMs: the bootloader EEPROM and the VL805 USB controller EEPROM. The bootloader EEPROM information is reported after the `BOOTLOADER` heading: here the `CURRENT` release is what is installed in this Pi's EEPROM and is the 3 September 2020 release, so this Pi can do USB mass storage boot since it has the required version. The `LATEST` version shown is the most recent version available from the [APT](https://www.raspberrypi.org/documentation/linux/software/apt.md) package present on this Pi.
+ 
+ If your Pi has an earlier release of bootloader EEPROM, there are two ways to upgrade to a version that can do USB mass storage boot:
+ 
+ 1. Use the `Misc Utility Images` option in the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to create an SD card with the latest `Raspberry Pi 4 EEPROM boot recovery` image. Once you have imaged the SD card, boot the Pi with this card and a display connected to one (or both) of the HDMI ports. As the Pi boots, its bootloader EEPROM will be upgraded to the latest release. The display will turn green to indicate a successful update.
+ 1. Run Raspberry Pi OS 2020-08-20 or later from an SD card, and use it to upgrade the EEPROM. First, upgrade the packages on your system using the command `sudo apt update` followed by `sudo apt upgrade`. Once you have done this, run `sudo rpi-eeprom-update`: it will present a `LATEST` version of the bootloader EEPROM from 3 September 2020 or later. Once you can see this new EEPROM version in the output of `rpi-eeprom-update`, reboot the Pi to install it.
```

Comment:
I believe you need to do _two_ reboots after the `sudo apt upgrade`? On the first reboot (of Linux) the eeprom-update-service will spot there's a newer EEPROM image available and copy it to the boot partition, and on the second reboot (of the firmware) it'll install the updated EEPROM image?

Reply:
I think we should just direct people to the rpi-imager install approach here. It's much simpler than documenting rpi-eeprom-update.

Reply:
> I think we should just direct people to the rpi-imager install approach here. It's much simpler than documenting rpi-eeprom-update.

Agreed - fixed.